keps/sig-api-machinery/20180415-crds-to-ga.md

```diff
@@ -162,10 +163,81 @@ TODO: complete this list
 
 ### Scale Targets for GA
 
-* TODO quantify: Read/write latency of CRDs within X% of native Kubernetes types
-* TODO quantify: Latency degrades less than X% for up to 100k Custom Resources per CRD kind
-* TODO quantify: Webhook conversion QPS of a noop converter is within X% of QPS with no webhook
-* Coordinate with sig-scalability
+The scale target for GA of custom resources are defined by the same [API call latency
+SLIs/SLOs as the Kuberetes native types](https://github.com/kubernetes/community/blob/master/sig-scalability/slos/api_call_latency.md#api-call-latency-slisslos-details).
+
+The targets are defined by the below suggested maximum limits, which are organized the same way as the [Kubernetes native type thresholds](https://github.com/kubernetes/community/blob/master/sig-scalability/configs-and-limits/thresholds.md#kubernetes-thresholds), with only one change:
+
+- Since custom resources can be arbitrarily large, we have broken down the limit by custom resource object size.
+
+**Custom Resources per Definition:**
+
+| Object size | Suggested Maximum Limit: scope=namespace (5s p99 SLO) | Suggested Maximum Limit: scope=cluster (30s p99 SLO) |
+| --- | --- | --- |
+| 10kb | 1500 | 10000 |
+| 25kb | 600 | 4000 |
+| 50kb | 300 | 2000 |
+
+To be consistent with Kubernetes native type SLOs, this SLO includes client serialization/deserialization.
```

Comment:
Technicall this isn't completely true - we're waiting only for Write() operation to finish (which translates to getting tcp ack), but doesn't necessary mean deserialization on the client side already happened.
Serialization client-side also isn't included.

Reply:
I would simply remove this sentence.

Reply:
Removed, I'll let the SLO doc speak for itself.